### PR TITLE
Adds CiviForm version to login page.

### PR DESCRIPTION
**Pull request by avrittrohwer**
### Description

Adds the CiviForm version to the login footer.  Feature is enabled by setting SHOW_CIVIFORM_IMAGE_TAG_ON_LANDING_PAGE=true in the civiform server environment variables.

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [x] Add new FeatureFlag env vars to `server/conf/application.conf`
- [x] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### CODE REVIEW

**Comment by bion** [comment]
@avrittrohwer this will look pretty ugly in prod with something like `CiviForm version: SNAPSHOT-k4jh3k-1231235` showing up on the page. Also we're already injecting this into a meta tag.

**Comment by avrittrohwer** [comment]
@bion the plan is to followup with an additional PR in civiform/cloud-deploy-infra that sets CIVIFORM_IMAGE_TAG to the version in the civiform_config.sh file.  So if someone is deploying v1.13.0, both this new element and the meta tag will have v1.13.0 instead of the commit tag

File: browser-test/src/landing_page.test.ts [hunk 1]
```diff
@@ -1,23 +1,65 @@
...
+    await validateScreenshot(ctx.page, screenshotName)
   })
 
   it('has no accessibility violations', async () => {
     await validateAccessibility(ctx.page)
   })
+}
+
+describe('the landing page', () => {
+  const ctx = createTestContext()
```
Comment (bion) [inline browser-test/src/landing_page.test.ts:30]: **suggestion:** 
```suggestion
  const {page} = createTestContext()
```

Comment (avrittrohwer) [reply browser-test/src/landing_page.test.ts]: Reverted the test

**Review by bion — APPROVED** [review]

**Pull request merged**